docs/operating-scylla/security/certificate-authentication.rst

```diff
@@ -0,0 +1,52 @@
+Certificate Based Authentication
+================================
+
+Certificate based authentication is an alternative method of resolving a login to credentials, i.e. a defined ScyllaDB role.
+Instead of defining one or more user/password pairs, each defined role is assigned depending on verified transport 
+client certificate Subject/Alt information.
```

Comment:
"verified transport 
client certificate Subject/Alt information." - Is that lever of detail required for any reason? Could we just say "verified 
client certificate information"?

Reply:
Perhaps overly specific.